paf-mvp-frontend/tests/helpers/cookies.ts

```diff
@@ -0,0 +1,48 @@
+import { Cookies } from '@core/cookies';
+import { Identifiers } from '@core/model/generated-model';
+
+const timestamp = Date.now();
+
+export const getFakePreferences = (consent = true) => ({
```

Comment:
I think it should at least be typed (`: Preferences`) to make sure the mock helper evolves with new versions of the model.